src/vivarium/testing_utilities.py

```diff
@@ -148,31 +149,62 @@ def _non_crn_build_population(
     return population
 
 
-def build_table(value, year_start, year_end, columns=("age", "year", "sex", "value")):
-    value_columns = columns[3:]
+def build_table(
+    value,
+    parameter_columns={
+        "age": (0, 125),
```

Comment:
Are these hard-coded defaults defined somewhere? If not I think they should be (_especially_ b/c 2020 will change basically every year)

Reply:
They are just arbitrary values based relatively on what our normal data looks like. No real data is compared to this these are just values used in this brute force approach to make this dataframe.